app/ui-flextab/client/tabs/inviteUsers.js

```diff
@@ -138,9 +138,7 @@ Template.inviteUsers.onCreated(function() {
 	this.ac = new AutoComplete({
 		selector: {
 			item: '.rc-popup-list__item',
-			container: '.rc-popup-list__list',
```

Comment:
Can you explain why are these changes needed? The PR seems to work fine without this.

Reply:
Sure. It was just a mistake. In the rush to submit a new fix, I chose the wrong branch and mixed the solutions to two different bugs. Now the fix to #14796 is the only remaining commit.